### PR TITLE
add csv export

### PR DESCRIPTION
**Pull request by lukasschreiber**

### CODE REVIEW

File: js/src/main/scala/webapp/Utils.scala [hunk 1]
```diff
@@ -48,3 +48,8 @@ def toQueryParameterName(in: String) = {
 def remToPx(rem: Float): Float = {
   rem * "^\\d*".r.findFirstIn(window.getComputedStyle(document.documentElement).fontSize).getOrElse("16").toFloat
 }
+
+def escapeCSVString(in: String): String = {
+  if (!"""\s|,|\"|(\r\n|\r|\n)""".r.findFirstMatchIn(in).isEmpty) s"\"$in\""
```
Comment (Oshgnacknak) [inline js/src/main/scala/webapp/Utils.scala:53]: we also need to escape quotes

**Review by Oshgnacknak — CHANGES_REQUESTED** [review]

File: js/src/main/scala/webapp/Utils.scala (hunk 1 above)
Comment (lukasschreiber) [reply js/src/main/scala/webapp/Utils.scala:53]: I thought a quoted text "text" becomes ""text"" in the csv which should be covered by the regex

Comment (lukasschreiber) [reply js/src/main/scala/webapp/Utils.scala:53]: or is it like `some longer "Text" with a quote` should become to "some longer ""Text"" with a quote"?

Comment (lukasschreiber) [reply js/src/main/scala/webapp/Utils.scala:53]: 7.  If double-quotes are used to enclose fields, then a double-quote
       appearing inside a field must be escaped by preceding it with
       another double quote.  For example:

       "aaa","b""bb","ccc"

ah okay, will change

**Review by Oshgnacknak — APPROVED** [review]

**Pull request merged**